src/main/java/com/laserfiche/repository/api/clients/EntriesClient.java

```diff
@@ -96,13 +95,12 @@ CompletableFuture<ODataValueOfIListOfFieldValue> assignFieldValues(String repoId
      * @param autoRename    An optional query parameter used to indicate if the new document should be automatically
      *                      renamed if an entry already exists with the given name in the folder. The default value is false.
      * @param culture       An optional query parameter used to indicate the locale that should be used.
-     *                      The value should be a standard language tag.
-     * @param inputStream   An InputStream object to read the raw bytes for the file to be uploaded.
+     *                      The value should be a standard language tag. This may be used when setting field values with tokens.
+     * @param file          The file that will be uploaded.
      * @param requestBody   null
      * @return CompletableFuture<CreateEntryResult> The return value
      */
-    CompletableFuture<CreateEntryResult> importDocument(String repoId, Integer parentEntryId, String fileName,
-            Boolean autoRename, String culture, InputStream inputStream, PostEntryWithEdocMetadataRequest requestBody);
```

Comment:
Would you please pull the latest 1.x so to include the recent changes from my pr today (i.e. using InputStream instead of File, as the method parameter)?